fbw-a32nx/src/systems/instruments/src/PFD/SpeedIndicator.tsx

```diff
@@ -1285,30 +1285,16 @@ export class MachNumber extends DisplayComponent<{ bus: ArincEventBus }> {
 
   private machHysteresis = false;
 
-  private onGround = false;
-
   private mach = new Arinc429Word(0);
 
-  private fcuEisDiscreteWord2 = new Arinc429Word(0);
-
-  private leftMainGearCompressed = true;
-
-  private rightMainGearCompressed = true;
-
   private handleMachDisplay() {
     if (this.mach.value > 0.5) {
       this.machHysteresis = true;
     } else if (this.mach.value < 0.45) {
       this.machHysteresis = false;
     }
 
-    const stdBaro = this.fcuEisDiscreteWord2.bitValueOr(28, false) || this.fcuEisDiscreteWord2.isFailureWarning();
-    const lsDisplay = this.fcuEisDiscreteWord2.bitValueOr(22, false) || this.fcuEisDiscreteWord2.isFailureWarning();
-
-    const hideMachDisplay =
-      (!this.machHysteresis && this.mach.isNormalOperation()) ||
-      (!this.mach.isNormalOperation() && (this.onGround || stdBaro)) ||
-      lsDisplay;
+    const hideMachDisplay = !this.machHysteresis && this.mach.isNormalOperation();
```

Comment:
Missing functional test.
```suggestion
    const hideMachDisplay = !this.machHysteresis && (this.mach.isNormalOperation() || this.mach.isFunctionalTest());
```

Reply:
Thanks, should be sorted now